Sources/Rosetta/PlayMode/CardSets/UldumCardsGen.cpp

```diff
@@ -1730,6 +1730,7 @@ void UldumCardsGen::AddShamanNonCollect(std::map<std::string, CardDef>& cards)
 
 void UldumCardsGen::AddWarlock(std::map<std::string, CardDef>& cards)
 {
+    Power power;
     // ---------------------------------------- SPELL - WARLOCK

```

Comment:
Can you insert a empty line to separate the declaration of the local variable and the definition of the card power?